backends/qualcomm/tests/models.py

```diff
@@ -301,6 +342,17 @@ def forward(self, x):
         return x.reshape(1, 12)
 
 
+class ScaledDotProductAttention(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, query_layer, key_layer, value_layer):
+        attn_output = torch.nn.functional.scaled_dot_product_attention(
+            query_layer, key_layer, value_layer
```

Comment:
Are you not testing with varying other args including mask?

Reply:
I just test the functionality of sdpa for now. Maybe we would add more test sets incoming.